app/test_main.py

```diff
@@ -1,3 +1,33 @@
 from app.main import get_human_age
 
-# write your code here
+
+def if_ages_are_zero() -> None:
```

Comment:
test name should have `test_` prefix
```suggestion
def test_ages_are_zero() -> None:
```

Reply:
fix everywhere